versions/1/kibana/spec.yml

```diff
@@ -83,3 +83,14 @@
           type: file
           contentMediaType: "application/json"
           pattern: '^{PACKAGE_NAME}-.+\.json$'
+    - description: Folder containing Kibana dashboard assets
+      type: folder
+      name: tag
+      required: false
+      contents:
+      - description: A dashboard tag file
+        type: file
+        contentMediaType: "application/json"
+        pattern: '^{PACKAGE_NAME}-.+\.json$'
+        forbiddenPatterns:
+          - '^.+-(ecs|ECS)\.json$' # ECS suffix is forbidden
```

Comment:
I think you can drop these properties. The `ecs` prefix was present for resources migrated from Beats.